main.go

```diff
@@ -1388,8 +1388,12 @@ func viewMatches(c *gin.Context) {
 	var err error
 	run := c.Param("run")
 	run = strings.TrimPrefix(run, "/")
-	if run == "" {
+	// 100 is a magic number used to limit initial page size
```

Comment:
Rather than using run - do something like up on line 1348 and use a query parameter to control whether to show all or not.

Reply:
Yes, having a separate queryparam is much better than using a magic number. Thanks, adjusted.